client/src/parsec/workspace.ts

```diff
@@ -466,5 +466,5 @@ export async function getSystemPath(
   if (!needsMocks()) {
     return await libparsec.mountpointToOsPath(infoResult.value.mountpoints[0][0], entryPath);
   }
-  return { ok: true, value: '/home' };
+  return { ok: true, value: `/home${entryPath}` };
```

Comment:
is entryPath always starting by a '/'?

Reply:
It doesn't matter much since this is just for the mocks, but yeah, it's a `FsPath` so it should always start with a `/` if we intend to give it to the bindings.